src/os/bluestore/BlueStore.h

```diff
@@ -569,10 +573,13 @@ class BlueStore : public ObjectStore,
 
   /// a cache (shard) of onodes and buffers
   struct Cache {
+    BlueStore *store;
```

Comment:
Seems you need that for accessing BlueStore::logger. Perhaps it's better to have direct pointer to logger here instead.